rpm/luacheck.spec

```diff
@@ -0,0 +1,41 @@
+Name: luacheck
+Version: 0.0.0
+Release: 1%{?dist}
+Summary: A static analyzer and a linter for Lua
+Group: Development/Tools
+License: MIT
+URL: https://github.com/mpeterv/luacheck
+Source0: luacheck-%{version}.tar.gz
+
+BuildArch: noarch
+
+BuildRequires: tarantool >= 1.7.2.0
+Requires: tarantool >= 1.7.2.0
+
+%description
+Luacheck is a command-line tool for linting and static analysis of Lua code.
+It is able to spot usage of undefined global variables, unused local variables
+and a few other typical problems within Lua programs.
+
+%prep
+%setup -q -n luacheck-%{version}
+
+%install
+mv ./bin/luacheck.lua luacheck.lua
+tarantool install.lua . %{_prefix}/share/tarantool/luacheck
+mkdir luacheck
+cp -r ./src/luacheck/* ./luacheck
```

Comment:
> mkdir luacheck
> cp -r ./src/luacheck/* ./luacheck

Is equivalent to the following?

> mv ./src/luacheck .

Reply:
Yes, it is. Thank you, now it's more simple.